### ISSUE TITLE
Port 25 blocked by new ISP: Mailu is not working anymore

### ISSUE THREAD

**Issue by leolivier** [Mailu/Mailu#1725]
Hi,
I'm using mailu since quite a long time with success but I just changed my ISP and the new one is blocking the port 25.
Now, even though my clients (Thunderbird or other on Android) are only using STARTLS or SSL/TLS (ports 587 & 465 for SMTP, 993 or 143 for IMAP), I can't send  nor receive mails anymore.
When I send an email to my address, I get after a few hours the message:
 `
<my address>: connect to <my mail domain>[IP address]:25:  Connection timed out
`
I looked at /conf/master.cf which only provides an SMTP service like this:
```
pi@raspberrypi /docker/mailu $ docker exec mailu.smtp more /conf/master.cf
# service type  private unpriv  chroot  wakeup  maxproc command + args
#               (yes)   (yes)   (yes)   (never) (100)

# Exposed SMTP service
smtp      inet  n       -       n       -       -       smtpd

# Internal SMTP service
10025     inet  n       -       n       -       -       smtpd
  -o smtpd_sasl_auth_enable=yes
  -o smtpd_client_restrictions=reject_unlisted_sender,reject_authenticated_sender_login_mismatch,permit
  -o smtpd_reject_unlisted_recipient={% if REJECT_UNLISTED_RECIPIENT %}{{ REJECT_UNLISTED_RECIPIENT }}{% else %}no{% endif %}
  -o cleanup_service_name=outclean
outclean  unix n       -       n       -       0       cleanup
  -o header_checks=pcre:/etc/postfix/outclean_header_filter.cf

# Internal postfix services
pickup    unix  n       -       n       60      1       pickup
cleanup   unix  n       -       n       -       0       cleanup
qmgr      unix  n       -       n       300     1       qmgr
tlsmgr    unix  -       -       n       1000?   1       tlsmgr
rewrite   unix  -       -       n       -       -       trivial-rewrite
bounce    unix  -       -       n       -       0       bounce
defer     unix  -       -       n       -       0       bounce
trace     unix  -       -       n       -       0       bounce
verify    unix  -       -       n       -       1       verify
flush     unix  n       -       n       1000?   0       flush
proxymap  unix  -       -       n       -       -       proxymap
smtp      unix  -       -       n       -       -       smtp
relay     unix  -       -       n       -       -       smtp
error     unix  -       -       n       -       -       error
retry     unix  -       -       n       -       -       error
discard   unix  -       -       n       -       -       discard
lmtp      unix  -       -       n       -       -       lmtp
anvil     unix  -       -       n       -       1       anvil
scache    unix  -       -       n       -       1       scache
postlog   unix-dgram n  -       n       -       1       postlogd

```
I tried a lot of things to add an smtps service, for instance adding a file named overrides/postfix.master and containing:
`smtp/inet=smtps inet n - - - - smtpd -o smtpd_tls_wrappermode=yes -o smtpd_sasl_auth_enable=yes -o smtpd_client_restrictions=permit_sasl_authenticated,reject -o milter_macro_daemon_name=ORIGINATING
`
but still it does not work.

So I'm totally lost, I don't see what I should do and I cannot find anything in Mailu documentation about that...
Thanks for any help

EDIT: Precision: I can connect to the server when I'm sending an email from my client and I can connect also when trying to receive

**Comment by leolivier** [comment]
When I try to send a message to another address, I get this in the postfix logs:
```
Jan 09 11:30:54 mail postfix/qmgr[340]: E65961EA027C: from=[my mailu address](mailto:my address), size=571, nrcpt=1 (queue active)
Jan 09 11:31:24 mail postfix/smtp[1484]: connect to mx1.free.fr[212.27.48.7]:25: Operation timed out
Jan 09 11:31:54 mail postfix/smtp[1484]: connect to mx1.free.fr[212.27.48.6]:25: Operation timed out
Jan 09 11:32:24 mail postfix/smtp[1484]: connect to mx2.free.fr[212.27.42.59]:25: Operation timed out
Jan 09 11:32:54 mail postfix/smtp[1484]: connect to mx2.free.fr[212.27.42.58]:25: Operation timed out
Jan 09 11:32:54 mail postfix/smtp[1484]: E65961EA027C: to=xxx@free.fr, relay=none, delay=61968, delays=61848/0.2/120/0, dsn=4.4.1, status=deferred (connect to mx2.free.fr[212.27.42.58]:25: Operation timed out)
```
Same if send an email to a gmail address:
```
Jan 09 11:45:54 mail postfix/qmgr[340]: EC6831EA167C: from=<my address>, size=595, nrcpt=1 (queue active)
Jan 09 11:45:55 mail postfix/smtp[1743]: connect to gmail-smtp-in.l.google.com[2a00:1450:400c:c06::1a]:25: Address not available
Jan 09 11:46:25 mail postfix/smtp[1743]: connect to gmail-smtp-in.l.google.com[66.102.1.26]:25: Operation timed out
Jan 09 11:46:55 mail postfix/smtp[1743]: connect to alt1.gmail-smtp-in.l.google.com[209.85.233.26]:25: Operation timed out
Jan 09 11:46:55 mail postfix/smtp[1743]: connect to alt1.gmail-smtp-in.l.google.com[2a00:1450:4010:c03::1b]:25: Address not available
Jan 09 11:46:55 mail postfix/smtp[1743]: connect to alt2.gmail-smtp-in.l.google.com[2404:6800:4003:c06::1b]:25: Address not available
Jan 09 11:46:55 mail postfix/smtp[1743]: EC6831EA167C: to=xxx@gmail.com, relay=none, delay=3449, delays=3388/0.15/60/0, dsn=4.4.1, status=deferred (connect to alt2.gmail-smtp-in.l.google.com[2404:6800:4003:c06::1b]:25: Address not available)

```
with the weird message about address not available

**Comment by leolivier** [comment]
Ok, forget, I managed to unblock the port 25, there was a firewall in the st top box of the ISP that blocked it. Things are working again... 
Anyway, it was a dumb question, it cannot work without port 25 it seems...
